index.html

```diff
@@ -10,224 +10,293 @@
+        <path
+          d="M18.4042 1.88042L20 0.337684V0H14.4717L10.5317 9.90989L6.04917 0H0.2525V0.337684L2.11667 2.60547C2.29833 2.77305 2.39333 3.01811 2.36917 3.26484V12.1768C2.42667 12.4977 2.32333 12.8278 2.1 13.0611L0 15.6328V15.9663H5.95417V15.6286L3.85417 13.0611C3.62667 12.8269 3.51917 12.5027 3.565 12.1768V4.46821L8.79167 15.9705H9.39917L13.8933 4.46821V13.6312C13.8933 13.8728 13.8933 13.9225 13.7367 14.0808L12.12 15.6615V16H19.9633V15.6623L18.405 14.1204C18.2683 14.016 18.1975 13.8408 18.2267 13.6707V2.33011C18.1975 2.15916 18.2675 1.984 18.4042 1.88042Z"
+          fill="#505F79" />
+      </svg>
+    </a><br></br>
+  </div>
+  <br></br>
+  <div>
+    <button id="bttn">Get My Resume</button>
+  </div><br>
+
+  <ul>
+    <div class="fr">
+      <p class="la">Languages </p>
+      <img class="dropdown" src="images/drop.jpeg" alt="">
     </div><br>
+
+    <li class="li">
+      <img src="shapes/Ellipse 1.png" alt="">
+      <p id="java">Javascript</p>
+    </li><br>
+
+    <li class="li">
+      <img src="shapes/eclipse/Ellipse 1.png" alt="">
+      <p id="htm">HTML</p>
+    </li><br>
+
+    <li class="li">
+      <img src="shapes/eclipse 2/Ellipse 1.png" alt="">
+      <P id="cs">CSS</P>
+    </li><br>
+
+  </ul>
+
+  <div > 
     <ul>
-      <div class="fr">
-        <p class="la">Languages </p>
-        <img class="dropdown" src="images/drop.jpeg" alt="">
-      </div><br>
-      <li class="li">
-        <img src="shapes/Ellipse 1.png" alt="">
-        <p id="java">Javascript</p>
-      </li><br>
-      <li class="li">
-        <img src="shapes/eclipse/Ellipse 1.png" alt="">
-        <p id="htm">HTML</p>
-      </li><br>
-      <li class="li">
-        <img src="shapes/eclipse 2/Ellipse 1.png" alt="">
-        <P id="cs">CSS</P>
-      </li><br>
+      <li class="fr">
+        <h1>
+          <p class="la">Frameworks </p>
+        </h1>
+        <img class="dropdown" src="images/dropdown.jpeg" alt="">
+      </li>
+      <hr />
+      <li class="fr">
+        <h1>
+          <p class="la">Skills</p>
+        </h1>
+        <img class="dropdown" src="images/dropdown.jpeg" alt="">
+      </li>
+      <hr />
     </ul>
-    <div>
-      <ul>
-        <li class="fr">
-          <h1>
-            <p class="la">Frameworks </p>
-          </h1>
-          <img class="dropdown" src="images/dropdown.jpeg" alt="">
-        </li>
-        <hr />
-        <li class="fr">
-          <h1>
-            <p class="la">Skills</p>
-          </h1>
-          <img class="dropdown" src="images/dropdown.jpeg" alt="">
-        </li>
-        <hr />
-      </ul>
-    </div>
-  </section>
+  </div>
+</section>
+
+<section class="contact">
+
+  <div>
+   <h2 class="contact-text">Contact me</h2>
+   <div class="par-container"><p class="par-contact">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p></div>
+  </div> 
```

Comment:
- [ ] Kindly consider slightly increasing the width of the text in the contact section. This will help to make the contact section match the Figma design 😻 

- Your design 😎 
![cont p](https://user-images.githubusercontent.com/99394374/213871662-ea720e0d-dc89-4ba5-9887-4cefb562c1df.PNG)